bitnami/cassandra/templates/statefulset.yaml

```diff
@@ -103,10 +123,14 @@ spec:
           volumeMounts:
             - name: data
               mountPath: {{ .Values.persistence.mountPath }}
+            {{- if .Values.persistence.commitStorageClass }}
+            - name: commitlog
+              mountPath: {{ .Values.persistence.commitLogMountPath }}
+        {{- end }}
```

Comment:
Could you put this line and the same level as the 'if'?

Reply:
Done